src/merger/models/mergerManager.ts

```diff
@@ -0,0 +1,49 @@
+import { injectable } from 'tsyringe';
+
+interface IExternalMappingModel {
+  tempOsmId: number;
+  externalId: string;
+}
+
+interface IOsmMappingModel {
```

Comment:
No need to start interface name with I, you only need it when the interface is an actual interface like in other OOP languages (when it meant to show common functionality).